extensions/adobe/experience/adcloud/adconversions.schema.json

```diff
@@ -0,0 +1,225 @@
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id": "https://ns.adobe.com/experience/adcloud/adconversions",
+  "title": "Raw Conversions for Advertising Platform",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:extends": ["https://ns.adobe.com/xdm/data/time-series"],
+  "description": "Raw Data that holds Cross Channel Advertising Conversion Events. ",
+  "definitions": {
+    "adconversions": {
+      "properties": {
+        "xdm:logId": {
+          "title": "Internal identifier for the batch",
+          "type": "number",
+          "description": "Internal identifier for the batch represented in the YYYYMMDDHH24 format"
+        },
+        "xdm:conversionTime": {
+          "title": "Conversion Time",
+          "type": "string",
+          "format": "date-time",
+          "description": "Timestamp that indicates when the conversion event occured"
+        },
+        "xdm:triggerTime": {
+          "title": "Trigger Time",
+          "type": "string",
+          "format": "date-time",
+          "description": "Timestamp that indicates when the click event occured"
+        },
+        "xdm:advertiserId": {
+          "title": "Advertiser Identifier",
+          "type": "string",
+          "description": "Advertiser ID for the campaign that initiated the conversion event"
+        },
+        "xdm:searchEngineId": {
+          "title": "Search Engine Identifier",
+          "type": "integer",
+          "description": "The application-specified identifier used to identify the Search Advertising Platform Name"
+        },
+        "xdm:propertyName": {
+          "title": "Property Name",
+          "type": "string",
+          "description": "Property Name Configured by Pixel, Analytics, Custom Feeds"
+        },
+        "xdm:clientId1": {
+          "title": "Client ID1",
+          "type": "string",
+          "description": "Alphanumeric strings that are matched against clicks to extract keyword,creative,adgroup IDs along with keyword text and matchtype"
+        },
+        "xdm:clientId2": {
+          "title": "Client ID2",
+          "type": "string",
+          "description": "Alphanumeric strings that are matched against clicks to extract keyword,creative,adgroup IDs along with keyword text and matchtype"
+        },
+        "xdm:accountId": {
+          "title": "Account Identifier",
+          "type": "string",
+          "description": "Account ID for the search advertising platform campaign that initiated the conversion event"
+        },
+        "xdm:campaignId": {
+          "title": "Campaign Identifier",
+          "type": "string",
+          "description": "Campaign ID on the advertising platform"
```

Comment:
campaignId for both search and dispaly. placementId is relevant to display only. I think we should change display to dsp since that is what was concluded in Wednesday's meeting